CoreUtilities/src/au/gov/asd/tac/constellation/utilities/genericjsonio/JsonIO.java

```diff
@@ -173,15 +183,34 @@ public static void saveJsonPreferences(final Optional<String> saveDir, final Opt
 
                 mapper.writeValue(preferenceFile, rootNode);
 
-                StatusDisplayer.getDefault().setStatusText(String.format(PREFERENCE_FILE_SAVED_MSG_FORMAT, 
+                StatusDisplayer.getDefault().setStatusText(String.format(PREFERENCE_FILE_SAVED_MSG_FORMAT,
                         preferenceFile.getPath()));
             } catch (final IOException ex) {
-                NotifyDisplayer.display(String.format("Can't save preference file: %s", ex.getMessage()), 
+                NotifyDisplayer.display(String.format(PREFERENCE_FILE_SAVE_ERROR, ex.getMessage()),
                         NotifyDescriptor.ERROR_MESSAGE);
+                LOGGER.log(Level.WARNING, String.format(PREFERENCE_FILE_SAVE_ERROR, prefixedFileName), ex);
             }
         }
     }
 
+    /**
+     * Get default keyboard shortcut. They are [Ctrl 1],[Ctrl 2],[Ctrl 3]...[Ctrl 5]     
+     * @param preferenceDirectory
+     * @return 
+     */
+    public static Optional<String> getDefaultKeyboardShortcut(final File preferenceDirectory) {
+        for (int index = 1; index <= 5; index++) {
+            final var fi = index;
```

Comment:
I think this variable can be removed and use `index` directly instead. If it is needed though, I'd prefer an explicit type used rather than var

Reply:
done